README.md

```diff
@@ -215,3 +215,55 @@ Currently, there are three policy configurations.
 
 A build Task, e.g. one that produces a container image, must abide by both `all-tasks` and
 `build-tasks` policy configurations.
+
+## Task Migration
+
+Task migrations are Bash scripts defined in version-specific task
+directories. In general, a migration consists of a series of `yq` commands that
+modify pipeline in order to work with the new version of task.
+
+Historically, task maintainers write `MIGRATION.md` to notify users what changes
+have to be made to the pipeline. This mechanism is not deprecated. Besides
+writing the file, it is also recommended to write a migration so that the
+updates can be applied to user pipelines automatically.
```

Comment:
It would also be good to mention how the migration files get applied - i.e. link to the migration tool and the configuration that enables it in our Renovate deployment  (Mintmaker)

Reply:
I was thinking we can just add a link to the migration tool, e.g.

> ...so that the updates can be applied to user pipelines automatically (by the [pipeline-migration-tool](https://github.com/konflux-ci/pipeline-migration-tool))

Reply:
Updated.